src/store/student-store.js

```diff
@@ -42,4 +42,10 @@ store.registerHandler('CHECKIN_SUCCESS', data => {
     store.emitChange();
 });
 
+
+store.registerHandler('ITEM_DUEDATE_UPDATED', data => {
+    student = data;
```

Comment:
What is this actually doing? If `data` has the items' new due dates, I think it is worth mentioning that in docs/Actions.md.